CIP-0015/test-vector.md

```diff
@@ -22,30 +22,30 @@ Catalyst **private** key hex
 Base address generated from payment key and staking key
```

Comment:
We should probably fix this too:
"~Base~ address generated from ~payment key and~ staking key"